pkg/walleterror/walleterror.go

```diff
@@ -49,6 +50,16 @@ func NewSystemError(module string, code int, errorName string, parentError error
 	}
 }
 
+// NewInvalidSDKUsageError creates a new invalid SDK usage error, used to indicate when a Wallet-SDK API is used
+// incorrectly.
+func NewInvalidSDKUsageError(module string, parentError error) *Error {
+	return &Error{
+		Code:        getErrorCode(module, incorrectUsageError, 0),
```

Comment:
I think it should be some big value like 10000 not 0, to not overlap with codes from the particular module.

Reply:
The error codes are relative to the category (incorrect usage), so overlap shouldn't be possible, right?

Reply:
i.e. they will always start with OCI3-